tests/ert_tests/gui/simulation/test_realization_view.py

```diff
@@ -1,6 +1,6 @@
 import pytest
 from qtpy import QtCore
-from qtpy.QtCore import QModelIndex, QPoint, QSize, Qt
+from qtpy.QtCore import QModelIndex, QSize
```

Comment:
In the future, you should either refrain from cleaning up imports in PRs like this one that involves a lot of files or you can have a separate commit with the cleanup changes